dev-cpp/cpptoml/files/cpptoml-0.1.1-limits.patch

```diff
@@ -0,0 +1,15 @@
+From c55a516e90133d89d67285429c6474241346d27a Mon Sep 17 00:00:00 2001
```

Comment:
Link to upstream PR / commit please at the top

Reply:
There doesn't seem to be an upstream commit for this, as the cpptoml project itself is dead with the last commit being from 6 years ago (which makes one wonder why Watchman started depending on it within the past year). Should I just remove the patch header instead?

Reply:
It's okay to keep it as-is (some provenance > none).